source/agora/test/Base.d

```diff
@@ -2320,7 +2320,7 @@ public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
       node : makeNodeConfig(),
       interfaces: [ makeInterfaceConfig("name.registry") ],
       consensus: makeConsensusConfig(),
-      network : net.nodes.map!(pair => Address("http://"~pair.address)).array.idup,
+      network : connect_addresses.filter!(addr => addr != Address("http://name.registry")).array.idup,
```

Comment:
Or we could just remove the registry from `full_node_addresses` (L2278)

Reply:
Let me see if that would break anything

Reply:
Done